src/main/resources/assets/html/partials/report/tdpr-report-details.html

```diff
@@ -31,7 +33,7 @@ <h1 id="report-header">Reports</h1>
 <div class="row options">
     <div class="col-md-12">
         <hr>
-        <h4>{{currentReportStart | date: DateFormat}} to {{currentReportEnd | date: DateFormat}}</h4>
+        <h4 class="navigate1 report-date">{{currentReportStart | date: 'dd.MM.yyyy'}} - {{currentReportEnd | date: 'dd.MM.yyyy'}}</h4>
```

Comment:
Could we change this css style name navigate1 to navigate without 1 in this issue?